proto/attestation/reference_value.proto

```diff
@@ -53,6 +53,24 @@ message BinaryReferenceValue {
   }
 }
 
+message KernelDigests {
+  // Contains admissible digests for the kernel image part of the kernel.
+  Digests image = 2;
+
+  // Contains admissible digests for the setup data part of the kernel.
+  Digests setup_data = 3;
+}
+
+// Follows the lines of BinaryReferenceValue, but provides a custom proto
+// to facilitate the digest matching.
+message KernelBinaryReferenceValue {
+  oneof type {
+    SkipVerification skip = 1;
+    EndorsementReferenceValue endorsement = 2;
+    KernelDigests digests = 3;
```

Comment:
Could / should this reuse the same proto as the endorsement? (probably not, but just checking)